framework/include/ble_manager/ble_client.h

```diff
@@ -59,6 +59,7 @@ typedef struct {
 	bool is_secured_connect;
 } ble_conn_info;
 
+#pragma pack(push, 4)
```

Comment:
As I know, for structure alignment in gcc, we should use `__attribute__ ((__packed__))`.
Please check this.

Reply:
And there is a trade-off for structure alignment.
If aligned, we can save the memory, but it takes much time as I know.
The purpose of this PR is for enhancing the speed, then we should check the trade-off.
Please check this.

Reply:
@JeonginKim 
https://developer.arm.com/documentation/dui0491/i/Compiler-specific-Features/-pragma-pack-n-

With ARM GCC compiler, we can use '#pragma'.

Reply:
When a structure is packed with 1 bytes, you are right.
As I know, the default value of packed size is 4 bytes. So there is no change with previous state.
I just added the packed size for ensuring the struct is multiple of 4

Reply:
Okay, understood. Thanks.